app/scripts/modules/core/src/presentation/forms/inputs/expression/ExpressionPreview.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Markdown } from 'core';
+
+export interface IExpressionPreviewProps {
+  spelPreview: string;
+  markdown: boolean;
+}
+
+export const ExpressionPreview = ({ spelPreview, markdown }: IExpressionPreviewProps) => (
+  <div className="flex-container-h baseline margin-between-lg">
```

Comment:
We can try using the new form selectors I added. https://github.com/spinnaker/styleguide/blob/master/src/styles/forms.css#L12

![image](https://user-images.githubusercontent.com/2153660/41257873-3c3e36b0-6d83-11e8-8232-201a6a4edf12.png)